.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "mantarray-magnet-finding"]
 	path = mantarray-magnet-finding
 	url = https://github.com/CuriBio/mantarray-magnet-finding.git
+[submodule "src/mantarray-magnet-finding"]
+	path = src/mantarray-magnet-finding
```

Comment:
Everything looks good, except I don't think the submodule is supposed to be in the `src` dir, the requirements.txt still has it in the root dir

Reply:
Hm, don't know why there were two references to submodule -- I'm removing the `./src/mantarray-magnet-finding` reference.